[2017-08-13 20:23:05] local.ERROR: Symfony\Component\Console\Exception\CommandNotFoundException: Command "dump-autoload" is not defined. in C:\xampp\htdocs\ERPOT\vendor\symfony\console\Application.php:515
Stack trace:
#0 C:\xampp\htdocs\ERPOT\vendor\symfony\console\Application.php(183): Symfony\Component\Console\Application->find('dump-autoload')
#1 C:\xampp\htdocs\ERPOT\vendor\symfony\console\Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 C:\xampp\htdocs\ERPOT\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 {main}  
[2017-08-13 20:27:15] local.ERROR: Symfony\Component\Console\Exception\CommandNotFoundException: Command "dump-autoload" is not defined. in C:\xampp\htdocs\ERPOT\vendor\symfony\console\Application.php:515
Stack trace:
#0 C:\xampp\htdocs\ERPOT\vendor\symfony\console\Application.php(183): Symfony\Component\Console\Application->find('dump-autoload')
#1 C:\xampp\htdocs\ERPOT\vendor\symfony\console\Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 C:\xampp\htdocs\ERPOT\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 {main}  
[2017-08-13 20:28:02] local.ERROR: Symfony\Component\Console\Exception\CommandNotFoundException: Command "seed" is not defined.

Did you mean one of these?
    db:seed
    make:seeder in C:\xampp\htdocs\ERPOT\vendor\symfony\console\Application.php:515
Stack trace:
#0 C:\xampp\htdocs\ERPOT\vendor\symfony\console\Application.php(183): Symfony\Component\Console\Application->find('seed')
#1 C:\xampp\htdocs\ERPOT\vendor\symfony\console\Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 C:\xampp\htdocs\ERPOT\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 {main}  
[2017-08-13 20:28:26] local.ERROR: PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'marenco_erp_training.trainings' doesn't exist in C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Connection.php:478
Stack trace:
#0 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Connection.php(478): PDO->prepare('delete from `tr...')
#1 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Connection.php(722): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'delete from `tr...', Array)
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Connection.php(685): Illuminate\Database\Connection->runQueryCallback('delete from `tr...', Array, Object(Closure))
#3 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Connection.php(483): Illuminate\Database\Connection->run('delete from `tr...', Array, Object(Closure))
#4 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Connection.php(438): Illuminate\Database\Connection->affectingStatement('delete from `tr...', Array)
#5 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(2202): Illuminate\Database\Connection->delete('delete from `tr...', Array)
#6 C:\xampp\htdocs\ERPOT\database\seeds\TrainingsTableSeeder.php(25): Illuminate\Database\Query\Builder->delete()
#7 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Seeder.php(39): TrainingsTableSeeder->run()
#8 C:\xampp\htdocs\ERPOT\database\seeds\DatabaseSeeder.php(36): Illuminate\Database\Seeder->call('TrainingsTableS...')
#9 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Console\Seeds\SeedCommand.php(63): DatabaseSeeder->run()
#10 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(2314): Illuminate\Database\Console\Seeds\SeedCommand->Illuminate\Database\Console\Seeds\{closure}()
#11 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Console\Seeds\SeedCommand.php(64): Illuminate\Database\Eloquent\Model::unguarded(Object(Closure))
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#14 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Console\Command.php(169): Illuminate\Container\Container->call(Array)
#15 C:\xampp\htdocs\ERPOT\vendor\symfony\console\Command\Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Console\Command.php(155): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 C:\xampp\htdocs\ERPOT\vendor\symfony\console\Application.php(794): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 C:\xampp\htdocs\ERPOT\vendor\symfony\console\Application.php(186): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 C:\xampp\htdocs\ERPOT\vendor\symfony\console\Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 C:\xampp\htdocs\ERPOT\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}

Next Illuminate\Database\QueryException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'marenco_erp_training.trainings' doesn't exist (SQL: delete from `trainings`) in C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Connection.php:729
Stack trace:
#0 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Connection.php(685): Illuminate\Database\Connection->runQueryCallback('delete from `tr...', Array, Object(Closure))
#1 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Connection.php(483): Illuminate\Database\Connection->run('delete from `tr...', Array, Object(Closure))
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Connection.php(438): Illuminate\Database\Connection->affectingStatement('delete from `tr...', Array)
#3 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Query\Builder.php(2202): Illuminate\Database\Connection->delete('delete from `tr...', Array)
#4 C:\xampp\htdocs\ERPOT\database\seeds\TrainingsTableSeeder.php(25): Illuminate\Database\Query\Builder->delete()
#5 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Seeder.php(39): TrainingsTableSeeder->run()
#6 C:\xampp\htdocs\ERPOT\database\seeds\DatabaseSeeder.php(36): Illuminate\Database\Seeder->call('TrainingsTableS...')
#7 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Console\Seeds\SeedCommand.php(63): DatabaseSeeder->run()
#8 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Eloquent\Model.php(2314): Illuminate\Database\Console\Seeds\SeedCommand->Illuminate\Database\Console\Seeds\{closure}()
#9 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Database\Console\Seeds\SeedCommand.php(64): Illuminate\Database\Eloquent\Model::unguarded(Object(Closure))
#10 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#11 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(507): call_user_func_array(Array, Array)
#12 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Console\Command.php(169): Illuminate\Container\Container->call(Array)
#13 C:\xampp\htdocs\ERPOT\vendor\symfony\console\Command\Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#14 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Console\Command.php(155): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#15 C:\xampp\htdocs\ERPOT\vendor\symfony\console\Application.php(794): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 C:\xampp\htdocs\ERPOT\vendor\symfony\console\Application.php(186): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 C:\xampp\htdocs\ERPOT\vendor\symfony\console\Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 C:\xampp\htdocs\ERPOT\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 {main}  
[2017-08-13 20:48:32] local.ERROR: RuntimeException: No supported encrypter found. The cipher and / or key length are invalid. in C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php:45
Stack trace:
#0 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('4CUzLsWAfXC/sJN...', 'AES-256-CBC')
#1 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(731): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build(Object(Closure), Array)
#3 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(849): Illuminate\Foundation\Application->make('encrypter')
#5 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(804): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(773): Illuminate\Container\Container->getDependencies(Array, Array)
#7 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#9 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(126): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(726): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(699): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#21 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#23 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#28 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\ERPOT\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#30 {main}  
[2017-08-13 20:48:32] local.ERROR: RuntimeException: No supported encrypter found. The cipher and / or key length are invalid. in C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php:45
Stack trace:
#0 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('4CUzLsWAfXC/sJN...', 'AES-256-CBC')
#1 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(731): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build(Object(Closure), Array)
#3 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(849): Illuminate\Foundation\Application->make('encrypter')
#5 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(804): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(773): Illuminate\Container\Container->getDependencies(Array, Array)
#7 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#9 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(152): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#10 C:\xampp\htdocs\ERPOT\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#11 {main}  
[2017-08-13 20:48:48] local.ERROR: RuntimeException: No supported encrypter found. The cipher and / or key length are invalid. in C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php:45
Stack trace:
#0 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('4CUzLsWAfXC/sJN...', 'AES-256-CBC')
#1 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(731): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build(Object(Closure), Array)
#3 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(849): Illuminate\Foundation\Application->make('encrypter')
#5 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(804): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(773): Illuminate\Container\Container->getDependencies(Array, Array)
#7 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#9 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(126): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(726): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(699): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#21 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#23 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#28 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\ERPOT\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#30 {main}  
[2017-08-13 20:48:48] local.ERROR: RuntimeException: No supported encrypter found. The cipher and / or key length are invalid. in C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php:45
Stack trace:
#0 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('4CUzLsWAfXC/sJN...', 'AES-256-CBC')
#1 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(731): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build(Object(Closure), Array)
#3 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(849): Illuminate\Foundation\Application->make('encrypter')
#5 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(804): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(773): Illuminate\Container\Container->getDependencies(Array, Array)
#7 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#9 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(152): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#10 C:\xampp\htdocs\ERPOT\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#11 {main}  
[2017-08-13 20:49:15] local.ERROR: RuntimeException: No supported encrypter found. The cipher and / or key length are invalid. in C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php:45
Stack trace:
#0 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('4CUzLsWAfXC/sJN...', 'AES-256-CBC')
#1 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(731): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build(Object(Closure), Array)
#3 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(849): Illuminate\Foundation\Application->make('encrypter')
#5 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(804): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(773): Illuminate\Container\Container->getDependencies(Array, Array)
#7 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#9 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(126): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(726): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(699): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#21 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#23 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#28 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\ERPOT\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#30 {main}  
[2017-08-13 20:49:15] local.ERROR: RuntimeException: No supported encrypter found. The cipher and / or key length are invalid. in C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php:45
Stack trace:
#0 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('4CUzLsWAfXC/sJN...', 'AES-256-CBC')
#1 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(731): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build(Object(Closure), Array)
#3 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(849): Illuminate\Foundation\Application->make('encrypter')
#5 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(804): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(773): Illuminate\Container\Container->getDependencies(Array, Array)
#7 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#9 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(152): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#10 C:\xampp\htdocs\ERPOT\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#11 {main}  
[2017-08-13 20:49:16] local.ERROR: RuntimeException: No supported encrypter found. The cipher and / or key length are invalid. in C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php:45
Stack trace:
#0 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('4CUzLsWAfXC/sJN...', 'AES-256-CBC')
#1 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(731): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build(Object(Closure), Array)
#3 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(849): Illuminate\Foundation\Application->make('encrypter')
#5 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(804): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(773): Illuminate\Container\Container->getDependencies(Array, Array)
#7 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#9 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(126): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#10 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#11 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#14 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(726): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#15 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(699): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#17 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#18 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#20 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#21 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#22 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#23 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#27 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#28 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\ERPOT\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#30 {main}  
[2017-08-13 20:49:16] local.ERROR: RuntimeException: No supported encrypter found. The cipher and / or key length are invalid. in C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php:45
Stack trace:
#0 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Encryption\EncryptionServiceProvider.php(25): Illuminate\Encryption\EncryptionServiceProvider->getEncrypterForKeyAndCipher('4CUzLsWAfXC/sJN...', 'AES-256-CBC')
#1 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(731): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build(Object(Closure), Array)
#3 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('encrypter', Array)
#4 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(849): Illuminate\Foundation\Application->make('encrypter')
#5 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(804): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#6 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(773): Illuminate\Container\Container->getDependencies(Array, Array)
#7 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Container\Container.php(629): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(697): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#9 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(152): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#10 C:\xampp\htdocs\ERPOT\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#11 {main}  
[2017-08-13 20:49:35] local.ERROR: InvalidArgumentException: View [Users.dashboard] not found. in C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:137
Stack trace:
#0 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(79): Illuminate\View\FileViewFinder->findInPaths('Users.dashboard', Array)
#1 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\View\Factory.php(165): Illuminate\View\FileViewFinder->find('Users.dashboard')
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\helpers.php(779): Illuminate\View\Factory->make('Users.dashboard', Array, Array)
#3 C:\xampp\htdocs\ERPOT\app\Http\Controllers\UsersViewsController.php(23): view('Users.dashboard')
#4 [internal function]: App\Http\Controllers\UsersViewsController->indexDashboard()
#5 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Controller.php(80): call_user_func_array(Array, Array)
#6 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(146): Illuminate\Routing\Controller->callAction('indexDashboard', Array)
#7 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(94): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\UsersViewsController), Object(Illuminate\Routing\Route), 'indexDashboard')
#8 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#9 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#11 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(96): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#13 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(54): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\UsersViewsController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'indexDashboard')
#14 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Route.php(174): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'indexDashboard')
#15 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Route.php(140): Illuminate\Routing\Route->runController(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(724): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\ERPOT\app\Http\Middleware\App.php(36): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 [internal function]: App\Http\Middleware\App->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#22 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(64): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#26 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#27 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#30 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#31 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#32 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(64): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#35 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#36 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#37 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#38 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#39 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(37): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#40 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#41 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#42 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#43 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#44 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(59): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#45 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#46 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#47 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#48 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#49 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#50 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#51 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(726): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#52 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(699): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#53 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#54 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#55 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#56 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#57 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#58 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#59 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#60 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#61 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#62 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#63 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#64 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#65 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#66 C:\xampp\htdocs\ERPOT\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#67 {main}  
[2017-08-13 20:49:35] local.ERROR: ErrorException: file_put_contents(C:\xampp\htdocs\ERPOT\storage\framework/sessions/b91dfed268ed639e2e3f9e3d1e880feeab0b472f): failed to open stream: No such file or directory in C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Filesystem\Filesystem.php:111
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'file_put_conten...', 'C:\\xampp\\htdocs...', 111, Array)
#1 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Filesystem\Filesystem.php(111): file_put_contents('C:\\xampp\\htdocs...', 'a:6:{s:6:"_toke...', 2)
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Session\FileSessionHandler.php(83): Illuminate\Filesystem\Filesystem->put('C:\\xampp\\htdocs...', 'a:6:{s:6:"_toke...', true)
#3 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Session\Store.php(262): Illuminate\Session\FileSessionHandler->write('b91dfed268ed639...', 'a:6:{s:6:"_toke...')
#4 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(88): Illuminate\Session\Store->save()
#5 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(155): Illuminate\Session\Middleware\StartSession->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#6 C:\xampp\htdocs\ERPOT\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#7 {main}  
[2017-08-13 20:49:36] local.ERROR: InvalidArgumentException: View [Users.dashboard] not found. in C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:137
Stack trace:
#0 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(79): Illuminate\View\FileViewFinder->findInPaths('Users.dashboard', Array)
#1 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\View\Factory.php(165): Illuminate\View\FileViewFinder->find('Users.dashboard')
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\helpers.php(779): Illuminate\View\Factory->make('Users.dashboard', Array, Array)
#3 C:\xampp\htdocs\ERPOT\app\Http\Controllers\UsersViewsController.php(23): view('Users.dashboard')
#4 [internal function]: App\Http\Controllers\UsersViewsController->indexDashboard()
#5 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Controller.php(80): call_user_func_array(Array, Array)
#6 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(146): Illuminate\Routing\Controller->callAction('indexDashboard', Array)
#7 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(94): Illuminate\Routing\ControllerDispatcher->call(Object(App\Http\Controllers\UsersViewsController), Object(Illuminate\Routing\Route), 'indexDashboard')
#8 [internal function]: Illuminate\Routing\ControllerDispatcher->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#9 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#10 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#11 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(96): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#13 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\ControllerDispatcher.php(54): Illuminate\Routing\ControllerDispatcher->callWithinStack(Object(App\Http\Controllers\UsersViewsController), Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'indexDashboard')
#14 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Route.php(174): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request), 'App\\Http\\Contro...', 'indexDashboard')
#15 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Route.php(140): Illuminate\Routing\Route->runController(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(724): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\ERPOT\app\Http\Middleware\App.php(36): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 [internal function]: App\Http\Middleware\App->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#22 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\VerifyCsrfToken.php(64): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#26 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#27 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#29 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\View\Middleware\ShareErrorsFromSession.php(49): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#30 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#31 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#32 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#34 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(64): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#35 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#36 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#37 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#38 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#39 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse.php(37): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#40 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#41 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#42 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#43 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#44 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Cookie\Middleware\EncryptCookies.php(59): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#45 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#46 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#47 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#48 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#49 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#50 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#51 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(726): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#52 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(699): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#53 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Router.php(675): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#54 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(246): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#55 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#56 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#57 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode.php(44): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#58 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#59 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(136): call_user_func_array(Array, Array)
#60 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#61 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#62 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#63 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Pipeline\Pipeline.php(102): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#64 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(132): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#65 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(99): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#66 C:\xampp\htdocs\ERPOT\public\index.php(53): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#67 {main}  
[2017-08-13 20:49:36] local.ERROR: ErrorException: file_put_contents(C:\xampp\htdocs\ERPOT\storage\framework/sessions/3f764675bd65af128aace7c6d382decc6d11bcaa): failed to open stream: No such file or directory in C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Filesystem\Filesystem.php:111
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'file_put_conten...', 'C:\\xampp\\htdocs...', 111, Array)
#1 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Filesystem\Filesystem.php(111): file_put_contents('C:\\xampp\\htdocs...', 'a:6:{s:6:"_toke...', 2)
#2 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Session\FileSessionHandler.php(83): Illuminate\Filesystem\Filesystem->put('C:\\xampp\\htdocs...', 'a:6:{s:6:"_toke...', true)
#3 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Session\Store.php(262): Illuminate\Session\FileSessionHandler->write('3f764675bd65af1...', 'a:6:{s:6:"_toke...')
#4 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Session\Middleware\StartSession.php(88): Illuminate\Session\Store->save()
#5 C:\xampp\htdocs\ERPOT\vendor\laravel\framework\src\Illuminate\Foundation\Http\Kernel.php(155): Illuminate\Session\Middleware\StartSession->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#6 C:\xampp\htdocs\ERPOT\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#7 {main}  
